app/views/layouts/angular/_multi_auth_credentials.html.haml

```diff
@@ -319,6 +319,13 @@
     miq_tabs_show_hide("#ssh_keypair_tab", true);
     miq_tabs_init('#auth_tabs');
     $('#auth_tabs').show();
+%div{"ng-if" => "#{ng_model}.emstype == 'vmware_cloud'"}
```

Comment:
@h-kataria Probably the simplest way is to add `vmware_cloud` controller into the list of those that configure tabs the same way, i.e. here

```
%div{"ng-if" => "#{ng_model}.emstype == 'ec2' || #{ng_model}.emstype == 'gce' || #{ng_model}.emstype == 'scvmm' || #{ng_model}.emstype == 'vmwarews'"}
```

However, we have another PR open for managing VMware events (#10192) where we enable the AMQP tab (https://github.com/xlab-si/manageiq/commit/d671c1184444dbaaa5a0b4f6813f2e7c7d3fb6d1#diff-19776d82dcdf8bc1080fe79eb95a254bR322). This will basically make `vmware_cloud` set tabs in the same way as OpenStack.

The question is whether I add `vmware_cloud` to the first list in this PR and update in the other one or should I add it to OpenStack already? I'd go with the former.

Is this what you had in mind with generic block?

/cc @miha-plesko